packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -336,35 +316,26 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 		);
 
 		itExpects(
-			"Receiving ops for swept datastores fails in client after sweep timeout and summarizing container",
+			"Ops for swept data stores is ignored but logs an error",
 			[
 				{
 					eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Loaded",
 					clientType: "noninteractive/summarizer",
 				},
 				{
-					eventName: "fluid:telemetry:ContainerRuntime:GC_Deleted_DataStore_Requested",
```

Comment:
Oops where these `_Requested` logs just wrong? Should have said `_Changed`, right?